src/components/ElementBar/ColorPickerButton/extractFirstNonTransparentOrFirstColor.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2024 Nordeck IT + Consulting GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Elements } from '../../../state/types';
+
+/**
+ * @param elements - Elements to search for a colour
+ * @returns If there are no elements it returns undefined.
+ *          Else the first non-transparent colour of the elements.
+ *          Finally 'transparent' if there is no non-transparent colour.
+ */
+export function extractFirstNonTransparentOrFirstColor(
```

Comment:
Please add a description to what this function is doing as it is done for other functions.

Currently it has `param` and `returns`, maybe some info from `returns` could go into description.

Reply:
Tried to rephrase it.